packages/webapp-yakshaving/package.json

```diff
@@ -24,12 +24,12 @@
 		"@rollup/plugin-replace": "^2.3.3",
 		"@rollup/plugin-sucrase": "^3.1.0",
 		"@rollup/plugin-typescript": "^6.0.0",
-		"rollup": "^2.33.2",
+		"rollup": "^2.33.3",
 		"rollup-plugin-terser": "^7.0.2",
 		"source-map-support": "^0.5.19",
-		"superstruct": "^0.10.12",
+		"superstruct": "^0.11.1",
```

Comment:
Reverting this to `0.10.12` should fix the errors.

Reply:
Good work 😅